tests/v2/validation/standalone/rke2/create_rancher_server/overlay/opt/corral/rke2/init-cluster.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+CORRAL_api_host="${CORRAL_fqdn}"
+echo "corral_set api_host=${CORRAL_api_host}"
+
+mkdir -p /etc/rancher/rke2
+cat > /etc/rancher/rke2/config.yaml <<- EOF
+write-kubeconfig-mode: 644
+cni: ${CORRAL_cni}
+tls-san:
+  - ${CORRAL_api_host}
+EOF
+
+# apt install -y jq || true
+apt install -y jq
```

Comment:
This only works for distros using apt (e.g. ubuntu). We can also hash this out later (since this problem also exists in the upstream packages), but another solution may be to download jq directly from upstream, or use a distro agnostic package manager like linuxbrew.

Reply:
Yes this exists in the upstream packages, definitely something we should look at. Using SLES it just ignores it as jq already exists on that distro